src/arrayMethodJoin.js

```diff
@@ -5,7 +5,46 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
```

Comment:
set default value for separator as
```suggestion
  [].__proto__.join2 = function(separator = ',') {
```